src/centos/scripts/start-marklogic.sh

```diff
@@ -341,6 +405,8 @@ fi
 if [[ -f /var/opt/MarkLogic/DOCKER_JOIN_CLUSTER ]]; then
     info "MARKLOGIC_JOIN_CLUSTER is true, but skipping join because this instance has already joined a cluster."
 elif [[ "${MARKLOGIC_JOIN_CLUSTER}" == "true" ]]; then
+    # Check if TLS is enabled on Bootstrap host
+    validate_tls_parameters "${BOOTSTRAP_STATUS}"
     # Validate bootsrap host before joining cluster
     BOOTSTRAP_STATUS=$(verify_bootstrap_status "${MARKLOGIC_BOOTSTRAP_HOST}")
```

Comment:
The verify_bootstrap_status has been called earlier. This looks like a duplication to me.

Reply:
BOOTSTRAP_STATUS=$(verify_bootstrap_status "${MARKLOGIC_BOOTSTRAP_HOST}") call is needed here in case where INIT=false and JOIN_CLUSTE=true is given, one of the examples is documented in the readme that uses docker-compose for configuring multi-node cluster.